kyaml/yaml/alias.go

```diff
@@ -19,6 +19,16 @@ const DefaultSequenceStyle = CompactSequenceStyle
 var sequenceIndentationStyle = DefaultSequenceStyle
 var indent = DefaultIndent
 
+// SetSequenceIndentationStyle sets the sequenceIndentationStyle variable
+func SetSequenceIndentationStyle(style string) {
+	sequenceIndentationStyle = style
+}
+
+// SequenceIndentationStyle returns the value of sequenceIndentationStyle
+func SequenceIndentationStyle() string {
+	return sequenceIndentationStyle
```

Comment:
looks like we should define new type `seqIndentType`

Reply:
Yes. Done.